### PR TITLE
Fix PHP unit tests

### PR DESCRIPTION
**Pull request by tn3rb**
looks like I forgot to add graphql settings to expected CPT config in our unit tests after merging the fast fetch branch into EDTR

### CODE REVIEW

**Comment by tn3rb** [comment]
going to merge this without further review since those were only linting fixes and we need this else no other EDTR branches will pass

**Comment by tn3rb** [comment]
@manzoorwanijk 

can you plz check your editor's formatting settings for a couple of things that were off in the files I fixed here:

- spaces must be used to indent lines; tabs are not allowed
- expected 1 newline at end of file
- array keys must be surrounded by spaces unless they contain a string or an integer

you can check your settings against all of the linting errors here if you like:
https://travis-ci.com/eventespresso/event-espresso-core/jobs/255730571

File: core/services/graphql/connections/ConnectionCollection.php [hunk 1]
```diff
@@ -52,8 +52,7 @@ private function loadCollection()
         if (! $this->loader instanceof CollectionLoader) {
             $this->loader = new CollectionLoader(
                 new CollectionDetails(
-                // collection name
-                    ConnectionCollection::COLLECTION_NAME,
+                    // collection nameConnectionCollection::COLLECTION_NAME,
```
Comment (manzoorwanijk) [inline core/services/graphql/connections/ConnectionCollection.php]: Messed up the args.

File: core/services/graphql/resolvers/ResolverCollection.php [hunk 2]
```diff
@@ -52,8 +52,7 @@ private function loadCollection()
         if (! $this->loader instanceof CollectionLoader) {
             $this->loader = new CollectionLoader(
                 new CollectionDetails(
-                // collection name
-                    ResolverCollection::COLLECTION_NAME,
+                    // collection nameResolverCollection::COLLECTION_NAME,
```
Comment (manzoorwanijk) [inline core/services/graphql/resolvers/ResolverCollection.php]: Here as well

File: core/services/graphql/types/TypeCollection.php [hunk 3]
```diff
@@ -52,8 +53,7 @@ private function loadCollection()
         if (! $this->loader instanceof CollectionLoader) {
             $this->loader = new CollectionLoader(
                 new CollectionDetails(
-                // collection name
-                    TypeCollection::COLLECTION_NAME,
+                    // collection nameTypeCollection::COLLECTION_NAME,
```
Comment (manzoorwanijk) [inline core/services/graphql/types/TypeCollection.php]: Same here

File: core/services/graphql/connections/ConnectionCollection.php (hunk 1 above)
Comment (tn3rb) [reply core/services/graphql/connections/ConnectionCollection.php]: OH YIKES!!!

good catch , will fix

File: core/services/graphql/resolvers/ResolverCollection.php (hunk 2 above)
Comment (tn3rb) [reply core/services/graphql/resolvers/ResolverCollection.php]: YIKES here as well !!

**Pull request merged**